src/maps/OverlayedMap.cpp

```diff
@@ -232,14 +250,13 @@ void OverlayedMap::drawAirport(const xdata::Airport& airport, double scale) {
     int px, py;
     positionToPixel(loc.latitude, loc.longitude, px, py);
 
-    bool hasControlTower = airport.hasTowerFrequency(); // ?
-    // bool hasControlTower = airport.hasMultipleATCFrequencies(); // Better than airport.hasTowerFrequency() ?
+    bool hasControlTower = airport.hasControlTower();
     uint32_t color = hasControlTower ? img::COLOR_ICAO_BLUE : img::COLOR_ICAO_MAGENTA;
     bool hasHardRunway = airport.hasHardRunway();
 
-    if (stitcher->getZoomLevel() <= 8) {
+    if (mapWidthNM > 200) {

```

Comment:
Could this be made a class constant intialized in the header? This will make it easier to re-use the constant in other places. Something like `static const int RUNWAY_DRAW_DIST_NM = 200`.